libs/designsystem/src/lib/components/tabs/tabs.component.spec.ts

```diff
@@ -17,7 +17,37 @@ describe('TabsComponent', () => {
     spectator = createHost({});
   });
 
-  it('should create', () => {
-    expect(spectator.component).toBeTruthy();
+  describe('hideTabBar: true', () => {
+    it('should hide tab bar', () => {
+      spectator.component.tabBarBottomHidden = true;
+      spectator.detectChanges();
+      const ionTabBarElm = spectator.query('ion-tab-bar');
+
+      expect(ionTabBarElm).toHaveComputedStyle({ display: 'none' });
+    });
+    it('should NOT set page footer safe area override if tab bar bottom is hidden', () => {
+      spectator.component.tabBarBottomHidden = true;
+      spectator.detectChanges();
+      const ionTabBarElm = spectator.query('ion-tab-bar');
+
+      expect(ionTabBarElm).toHaveComputedStyle({ '--kirby-page-footer-safe-area-bottom': '' });
+    });
+  });
+
+  describe('hideTabBar: false', () => {
+    it('should show tab bar', () => {
+      spectator.component.tabBarBottomHidden = false;
+      spectator.detectChanges();
+
+      const ionTabBarElm = spectator.query('ion-tab-bar');
+      expect(ionTabBarElm).not.toHaveComputedStyle({ display: 'none' });
+    });
+    it('should set footer safe area to 0 when tab bar is shown and in bottom', () => {
+      spectator.component.tabBarBottomHidden = false;
+      spectator.detectChanges();
+
+      const ionTabBarElm = spectator.element;
+      expect(ionTabBarElm).toHaveComputedStyle({ '--kirby-page-footer-safe-area-bottom': '0px' });
+    });
```

Comment:
This test is false positive - you are asserting the css var on the tabs element itself, but you should assert on the ion-tab-bar.
Also: When not explicitly setting a window size, you should assume desktop - by which the tabs are shown at top, and the safe-area footer override should be empty: 
```suggestion
    it('should set footer safe area to 0 when tab bar is shown and in bottom', () => {
      spectator.component.tabBarBottomHidden = false;
      spectator.detectChanges();

      const ionTabBarElm = spectator.query('ion-tab-bar');
      expect(ionTabBarElm).toHaveComputedStyle({ '--kirby-page-footer-safe-area-bottom': '' });
    });
```